Apps/WebClient/src/ClientApp/src/components/private/home/AddQuickLinkComponent.vue

```diff
@@ -276,79 +281,93 @@ function hideModal(): void {
-                        data-testid="organ-donor-registration-filter"
-                        name="organ-donor-registration-filter"
-                        value="organ-donor-registration"
-                        label="Organ Donor Registration"
-                        density="compact"
-                        hide-details
-                        color="primary"
-                    />
-                    <v-checkbox
-                        v-if="showHealthConnectRegistry"
-                        id="health-connect-registry-filter"
-                        v-model="selectedQuickLinks"
-                        data-testid="health-connect-registry-filter"
-                        name="health-connect-registry-filter"
-                        value="health-connect-registry"
-                        label="Health Connect Registry"
-                        density="compact"
-                        hide-details
-                        color="primary"
-                    />
-                    <v-checkbox
-                        v-if="showVaccineCard"
-                        id="bc-vaccine-card-filter"
-                        v-model="selectedQuickLinks"
-                        data-testid="bc-vaccine-card-filter"
-                        name="bc-vaccine-card-filter"
-                        value="bc-vaccine-card"
-                        label="BC Vaccine Card"
-                        density="compact"
-                        hide-details
-                        color="primary"
-                    />
-                    <v-checkbox
-                        v-if="showImmunizationRecord"
-                        id="immunization-record-filter"
-                        v-model="selectedQuickLinks"
-                        data-testid="immunization-record-filter"
-                        name="immunization-record-filter"
-                        value="immunization-record"
-                        label="Add Vaccines"
-                        density="compact"
-                        hide-details
-                        color="primary"
-                    />
-                    <v-checkbox
-                        v-if="showRecommendationsDialog"
-                        id="recommendations-dialog-filter"
-                        v-model="selectedQuickLinks"
-                        data-testid="recommendations-dialog-filter"
-                        name="recommendations-dialog-filter"
-                        value="recommendations-dialog"
-                        label="Vaccine Recommendations"
-                        density="compact"
-                        hide-details
-                        color="primary"
-                    />
+                    <v-chip-group v-model="selectedQuickLinks" column multiple>
+                        <v-chip
+                            v-for="quickLink in enabledQuickLinkFilter"
+                            :id="quickLink.module + '-filter'"
```

Comment:
Can't we use template literals like `${quickLink.module}-filter`?  Same for the data-testid and name.

Reply:
Sure thing, I let co-pilot generate this one, and that slipped through my review.